### PR TITLE
Handle WebAuthn challenge having multiple factors

### PR DESCRIPTION
**Pull request by scotttrinh**
We made some big changes late in the game for allowing multiple WebAuthn factors for a given (email, user_handle) pair, but didn't update this in the authentication challenge flow.

Since we cannot remove the old link, this adds an additional link and ignores the old one.

### CODE REVIEW

**Comment by msullivan** [comment]
I would kind of prefer if we *also* looked at the old link, but I guess it's not totally critical for something like this

**Review by jaclarke — APPROVED** [review]

**Comment by scotttrinh** [comment]
@msullivan 

Oh yeah, I remember we talked about that. Lemme see if it's easy to distinct union them together in the query so that everything else just keeps working.

**Comment by scotttrinh** [comment]
@msullivan ⚡ [`2af19e9` (#6945)](https://github.com/edgedb/edgedb/pull/6945/commits/2af19e9ec68c8e01ada3f9fbe760f533c02e5cf2)

Let me know if that makes sense!

**Comment by scotttrinh** [comment]
@msullivan ⚡ [`8fe06a2` (#6945)](https://github.com/edgedb/edgedb/pull/6945/commits/8fe06a2d61660f25e7ea7c6881666dfcf54d7a54)

As discussed, just removing the property.

File: edb/server/protocol/auth_ext/webauthn.py [hunk 1]
```diff
@@ -315,18 +315,19 @@ async def create_authentication_options_for_email(
...
             select ext::auth::WebAuthnFactor
             filter .user_handle = user_handle
             and .email = email
-        )))
+        ))
     )
 insert ext::auth::WebAuthnAuthenticationChallenge {
     challenge := challenge,
-    factor := factor,
+    factor := assert_exists(assert_single((select factors limit 1))),
```
Comment (jaclarke) [inline edb/server/protocol/auth_ext/webauthn.py:327]:
```suggestion
```

**Pull request merged**